### ISSUE TITLE
New Experimental updates 

### ISSUE THREAD

**Issue by 7Rickus** [LancelotP/satisfactory-map#39]
Hi Lancelot - are you still actively maintaining this project, or has it been abandoned?

I ask because of the new “Experimental” update, which has changed the nodes.

I love this tool, and would hate to see it abandoned. 

PS. I am a software engineer by trade, but mainly c++, and have no experience in typescript or web development. If you have abandoned this project, would you mind me making a stand-alone app based on your code?

**Comment by LancelotP** [comment]
Hello @7Rickus,

This project is not abandoned, just been really busy and lost a bit of motivation. I'm working on getting it back up ;).

I'm in the process of updating the nodes and adding dual version for live / exp versions ASAP.

Feel free to implement a standalone one by all mean as well :).

**Comment by 7Rickus** [comment]
That’s completely understandable. I’m glad you are still maintaining this project. It is (in my opinion) the very best of the map projects out there.

PS. Have you found any way of extracting ALL the nodes/item locations from the save game or the Pak file?

Cheers,
Rick

**Comment by LancelotP** [comment]
@7Rickus Yup, from the save games. The method did not change much since release but with each update the data change a bit so need to double check each time something is released ;).

New nodes are on the way for the exp build, need a bit more work will publish tomorrow
